Makefile

```diff
@@ -115,6 +114,11 @@ help: ## Default target, parses special in-line comments as documentation.
 	@grep -E '^[[:print:]]+:.*?## .*$$' $(MAKEFILE_LIST) | \
 		awk 'BEGIN {FS = ":(.*)?## "}; {printf $(_HLPFMT), $$1, $$2}'
 
+# N/B: There are length/character limitations in GCE for image names
+.PHONY: IMG_SFX
+IMG_SFX:  ## Generate a new date-based image suffix into the file IMG_SFX
+	$(file >$@,$(shell date +%y%m%d)v0-f$(FEDORA_RELEASE)p$(PRIOR_FEDORA_RELEASE)u$(subst .,,$(UBUNTU_RELEASE)))
```

Comment:
SGTM.  Looks like the limit is 63-characters.  Since the timestamp is being kept in a user-generated file, let me try adding hours/minutes/seconds as well.  That will avoid the whole "woops I overwrote your image" problem.